app/assets/javascripts/models/beta/sprint.js

```diff
@@ -0,0 +1,4 @@
+export const sort = sprints => 
+  sprints.every(sprint => isDone(sprint)) ? sprints.reverse() : sprints
+
+export const isDone = sprint => sprint.hasOwnProperty('hasStories');
```

Comment:
I would describe more the `sort` function, like `sortSprints`
Also, I would rewrite this file like this:
```js
import { every, has } from 'underscore';

export const isDone = sprint => has(sprint, 'hasStories');

const isAllDone = sprints => every(sprints, isDone);

export const sortSprints = sprints => 
  isAllDone(sprints) ? sprints.reverse() : sprints
```